### DISCUSSION TITLE
Change from yi-hack-Allwinner-v2 to yi-hack-v5

### DISCUSSION THREAD

**Discussion by gomble** [alienatedsec/yi-hack-v5#310] [Q&A]
Hello,

i flashed my YI Dome Guard r35gb with yi-hack-Allwinner-v2, but it is not stable. How can i change from yi-hack-Allwinner-v2 to yi-hack-v5?

**Comment by alienatedsec** [comment]
That will never work and it's better if you discuss the stability issues with @roleoroleo rather than trying something dedicated to an older camera generation.

**Reply by gomble** [reply]
Old camera standard??
It is marked here -> supported
![image](https://github.com/alienatedsec/yi-hack-v5/assets/6769445/07122882-aa3b-4e96-a601-eaad6a67a5e2)
And why should that not work???
If i factory reset the yi dome guard 1080p and flash yi-hack-v5 why should it not work??

**Reply by roleoroleo** [reply]
v5 supports camera based on Hisilicon SOC
MStar supports cams based on MStar SOC
Allwinner and Allwinner-v2 support cams based on Allwinner SOC

The hw is different and you can't use a cam with a fw based on a different platform.
The name of the cam is not enough to know which platform is, you need suffix and/or fw version.
For example "Yi 1080p Home" exists in all 3 platforms.

Please let me know which kind of problem your allwinner cam has.
And open a issue, if you haven't already done.
I will try to support you, if the problem has a solution.

**Reply by gomble** [reply]
i have posted my problem here: https://github.com/roleoroleo/yi-hack-Allwinner-v2/issues/732

But maybe the firmware is wrong? I didnt know that there are 3 different SOCs. Maybe Allwinner-v2 SOC is wrong? How can i check my SOC?

**Reply by roleoroleo** [reply]
If your cam works with r35gb it's an Allwinner-v2.
Allwinner and Allwinner-v2 use the same chip, but they have a different flash partition scheme andfile system structure.
